src/test/java/space/obminyashka/items_exchange/end2end/AuthorizationFlowTest.java

```diff
@@ -68,7 +68,8 @@ void register_shouldCreateValidNewUserAndReturnCreated() throws Exception {
     }
 
     @Test
-    void register_whenDtoIsValid_shouldReturnSpecificSuccessMessage() throws Exception {
+    @ExpectedDataSet(value = "auth/create_token.yml", ignoreCols = {"id", "created", "expiry_date", "token", "user_id"})
```

Comment:
The email_confirmation_token has 5 columns and here you're ignoring them all. Why that test should be count as valid?